tensorflow/tools/ci_build/install/install_python38.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+cd /usr/src
+wget https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tgz
+tar xzf Python-3.8.2.tgz
+cd Python-3.8.2
+./configure --enable-optimizations  --with-ensurepip=install --prefix=/usr/
+make altinstall
+rm /usr/src/Python-3.8.2.tgz
+rm -rf /usr/src/Python-3.8.2
```

Comment:
do we want to add a link so /usr/bin/python point to this new python3.8?

Reply:
It does not seem necessary. What is the benefit of letting /usr/bin/python point to python3.8?

Reply:
RUN update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 0 
This step was run in the docker file, so no need to do in this file.

Reply:
sounds good